src/gamepad/linux/linux-guid-helpers.vala

```diff
@@ -0,0 +1,31 @@
+// This file is part of GNOME Games. License: GPLv3
+
+private class LibGamepad.LinuxGuidHelpers : Object {
+	public static string from_dev (Libevdev.Evdev device) {
```

Comment:
This should go with `uint16s_to_hex_string` and clearly not in an empty class. I'm not sure what's the good name as it's both related to SDL2 mappings and Evdev... The namespace's name should reflect that.